draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -454,26 +442,23 @@ and can be sent in a trailer section.
 
 ## RateLimit-Policy {#ratelimit-policy-field}
 
-The `RateLimit-Policy` response field indicates
-the `quota` associated to the client
+The RateLimit-Policy field response field indicates
+the quota associated to the client
 and its value is informative.
 
-The field is a non-empty List of quota policies (see {{quota-policy}}).
+The field is a non-empty List of Items.
+Each item is a [quota policy](#quota-policy).
 
-~~~ abnf
-   RateLimit-Policy = sf-list
-~~~
-
-A `time-window` associated to `expiring-limit` can be communicated
-via `RateLimit-Policy`, like shown in the following example.
+This field can convey the time-window associated to the expiring-limit,
+like shown in the following example.
 
 ~~~ example
    RateLimit-Policy: 100;w=10
-   RateLimit-Limit: 100
+   RateLimit-Limit:  100
```

Comment:
Why the extra whitespace?

Reply:
to align with the response above. It is ok to remove though.